### PR TITLE
fixes #25404 - add command to recalculate errata for a host

### PR DESCRIPTION
**Pull request by jduepmeier**
This pull request adds a command to recalculate errata for a host.
It uses the /hosts/<host_id>/errata/applicability api endpoint.

### CODE REVIEW

**Comment by akofink** [comment]
Thanks @jduepmeier! Could you please file an issue [in redmine](https://projects.theforeman.org/projects/katello/issues/new) under the Katello project for this?

File: test/functional/host/errata/recalculate_test.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      'state' => 'stopped'
+    }
+  end
+
+  it "recalculates errata for a host" do
+    params = ["--host-id=#{host_id}"]
+
+    ex = api_expects(:host_errata, :applicability, 'Host errata recalculate') do |par|
+      par['host_id'] == host_id
+    end
```
Comment (akofink) [inline test/functional/host/errata/recalculate_test.rb]: I'd prefer the `with_params` syntax. It provides better output (shows all API params expected/actual) when the tests fail. It looks like this:

```rb
api_expects(:host_errata, :applicability, 'Host errata recalculate')
  .with_params('host_id' => host_id)
  .returns(response)
```

The same for the other test.

Comment (jduepmeier) [reply test/functional/host/errata/recalculate_test.rb]: I copied this test from the other test in the folder ([apply_test.rb](https://github.com/Katello/hammer-cli-katello/blob/master/test/functional/host/errata/apply_test.rb)).
But i can change this.

**Comment by jduepmeier** [comment]
Issue in redmine is [25404](https://projects.theforeman.org/issues/25404)

**Pull request merged**